src/octod/core.d

```diff
@@ -35,6 +35,14 @@ struct Configuration
     string oauthToken;
     /// By default client works in live mode
     bool dryRun = false;
+
+    /// Returns true if this is a github configuration
+    public bool isGithub ( ) const
```

Comment:
I know there are only two supported platforms but still it would be cleaner to have `PlatformKind platform()` method which returns enum value :)